pkg/_pub_shared/lib/validation/html/html_validation.dart

```diff
@@ -74,6 +74,20 @@ void validateHtml(Node root) {
     }
   }
 
+  // All <a href=""> links must be valid and have only single slashes.
+  for (final elem in links) {
```

Comment:
This will also be testing external links create by packages right, they can legitimately have double slashes...

Reply:
Just saying, I think we don't need this...